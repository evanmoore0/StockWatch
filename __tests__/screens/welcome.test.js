import React from "react";
import renderer from 'react-test-renderer'
import Welcome from "../../screens/welcome";

describe('<Welcome/>', () => {
    it('renders correctly',  () => {
        const tree = renderer.create(<Welcome/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})
