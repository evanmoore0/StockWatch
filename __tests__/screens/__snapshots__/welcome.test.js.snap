// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Welcome/> renders correctly 1`] = `
<RNCSafeAreaView
  style={
    Object {
      "flex": 1,
      "paddingTop": 28,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "width": 278,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 38,
          "fontWeight": "700",
        }
      }
    >
      Stock Watch
    </Text>
  </View>
  <View
    style={
      Object {
        "paddingTop": 7,
        "width": 278,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "gray",
          "fontSize": 17,
          "paddingBottom": 4,
          "textAlign": "center",
        }
      }
    >
      Keep track of all your stocks
    </Text>
    <Text
      style={
        Object {
          "color": "gray",
          "fontSize": 17,
          "paddingBottom": 4,
          "textAlign": "center",
        }
      }
    >
      at the click of a button
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <View
        style={
          Object {
            "top": 278,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 120,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 197,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 78,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 167,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 115,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 194,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#82C8FB",
              "borderRadius": 56,
              "height": 115,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 111,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 115,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 56,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 58,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 14,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 135,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 83,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#82C8FB",
              "borderRadius": 56,
              "height": 154,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 125,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#82C8FB",
              "borderRadius": 56,
              "height": 63,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 153,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#82C8FB",
              "borderRadius": 56,
              "height": 97,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 83,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 154,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 14,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 135,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": -41,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 58,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 0,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#82C8FB",
              "borderRadius": 56,
              "height": 78,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": 42,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#82C8FB",
              "borderRadius": 56,
              "height": 120,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": -41,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 120,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": -125,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 135,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": -208,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 115,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": -249,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 97,
              "width": 14,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "top": -305,
            "transform": Array [
              Object {
                "scale": 50,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "backgroundColor": "#6AB664",
              "borderRadius": 56,
              "height": 115,
              "width": 14,
            }
          }
        />
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "paddingBottom": 208,
          "width": "100%",
        },
        Object {
          "paddingBottom": 28,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#6AB664",
          "borderRadius": 70,
          "height": 56,
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 97,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 22,
          }
        }
      >
        Register
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "paddingBottom": 208,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 70,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 14,
          }
        }
      >
        Already have an account?
      </Text>
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 14,
          }
        }
      >
        Login here
      </Text>
    </View>
  </View>
</RNCSafeAreaView>
`;
